### PR TITLE
Add logging levels to custom plugin logs.

### PR DESCRIPTION
**Pull request by abansal4032**

### CODE REVIEW

File: pkg/custompluginmonitor/plugin/plugin.go [hunk 1]
```diff
@@ -234,9 +234,9 @@ func (p *Plugin) Stop() {
 	glog.Info("Stop plugin execution")
 }
 
-func logPluginStderr(path, logs string) {
+func logPluginStderr(rule cpmtypes.CustomRule, logs string, logLevel glog.Level) {
 	if len(logs) != 0 {
-		glog.Infof("Start logs from plugin %q \n %s", path, string(logs))
-		glog.Infof("End logs from plugin %q", path)
+		glog.V(logLevel).Infof("Start logs from plugin %q \n %s", rule, logs)
+		glog.V(logLevel).Infof("End logs from plugin %q", rule)
```
Comment (Random-Liu) [inline pkg/custompluginmonitor/plugin/plugin.go:240]: %q?

Comment (abansal4032) [reply pkg/custompluginmonitor/plugin/plugin.go]: %q will remove the formatting from the plugin logs and they will be less readable.

**Pull request merged**